kb/prometheus-grafana-setup-cloud.md

```diff
@@ -0,0 +1,253 @@
+
+To check whether Prometheus is receiving metrics from your SDK target, go to http://localhost:9090 and navigate to **Status&nbsp;> Targets**.
+The status of your target endpoint defined in your configuration appears here.
+
+## Data sources configuration for Temporal Cloud and SDK metrics in Grafana
+
+Depending on how you use Grafana, you can either install and run it locally, run it as a Docker container, or log in to Grafana Cloud to set up your data sources.
+
+If you have installed and are running Grafana locally, go to [http://localhost:3000](http://localhost:3000) and sign in.
+
+You must configure your Temporal Cloud and SDK metrics data sources separately in Grafana.
+
+To add the Temporal Cloud Prometheus HTTP API endpoint that we generated in the previous section, do the following:
+
+1. Go to **Configuration&nbsp;> Data sources**.
+2. Select **Add data source&nbsp;> Prometheus**.
+3. Enter a name for your Temporal Cloud metrics data source, such as _Temporal Cloud metrics_.
+4. In the **HTTP section**, paste the URL that was generated in the Observability section on the Temporal Cloud UI.
+5. In the **Auth section**, enable **TLS Client Auth**.
+6. In the **TLS/SSL Auth Details** section, paste the root CA certificate and key that you used to [set up observability](#temporal-cloud-metrics-setup) in your Temporal Cloud UI.
+7. Select **Save and test** to ensure the data source is working properly.
+
+If you see issues in setting this data source, verify that you are using the correct certificates (root CA) on both your Temporal Cloud observability setup and in the TLS authentication in Grafana.
+
+To add the SDK metrics Prometheus endpoint that we configured in the previous section, do the following:
+
+1. Go to **Configuration&nbsp;> Data sources**.
+2. Select **Add data source&nbsp;> Prometheus**.
+3. Enter a name for your Temporal Cloud metrics data source, such as _Temporal SDK metrics_.
+4. In the **HTTP** section, enter your Prometheus endpoint in the URL field.
+   If running Prometheus locally as described in the examples in this article, enter `http://localhost:9090`.
+5. For this example, enable **Skip TLS Verify** in the **Auth** section.
+6. Select **Save and test** to ensure that the data source is working properly.
+
+If you see issues in setting this data source, check whether the endpoints set in your SDKs are showing metrics.
+If you don’t see your SDK metrics at the scrape endpoints defined, check whether your Workers and Workflow Executions are running.
+If you see metrics on the scrape endpoints, but Prometheus shows your targets are down, then there is an issue with connecting to the targets set in your SDKs.
+Verify your Prometheus configuration and restart Prometheus.
+
+If you’re running Grafana as a container, you can set your SDK metrics Prometheus data source in your Grafana configuration.
+See the example Grafana configuration described in the Prometheus and Grafana setup for open-source Temporal Cluster.
```

Comment:
Is there a link to `Prometheus and Grafana setup for open-source Temporal Cluster`?